api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -546,6 +546,12 @@ type HostedClusterSpec struct {
 	// +optional
 	Configuration *ClusterConfiguration `json:"configuration,omitempty"`
 
+	// OperatorConfiguration specifies configuration for individual OCP operators in the cluster.
```

Comment:
please start doc with lower case so generate gdoc uses the serialized version of the field i.e. operatorConfiguration.
Same for the inner field.
FWIW we'll add linter for this.

Reply:
Thanks, the doc comments for the `operatorConfiguration` and `clusterVersionOperator` fields have been updated!